### PR TITLE
Updated README.md

### LINKED ISSUE #1013: Update README.md for JavaScript
The markdown file for the JavaScript directory  hasn't been updated. I would like to update the list alphabetically following the same structure as of the README. Please assign this issue to me.

### PR DESCRIPTION
**Pull request by ireneelizabethsabu**
# Description
Fixes #1013 

## Type of change
- [ ] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: JavaScript/README.md [hunk 1]
```diff
@@ -14,11 +15,17 @@
...
+- [Github Good First Issues extractor](Github_Good_First_Issues)
 - [Github Graph Populator](GitHub_Graph_Populator)
+- [Who unfollowed you on Github](Github_Unfollowers)
 - [HTML to Markdown](Html_to_Markdown)
 - [Like Posts automatically on Instagram](Instagram_Like)
 - [ROT13 Decryption](ROT13_Decryption)
 - [RSA Key Pair Generator](RSA_Key_Pair_Generator)
-- [Send Emails](Send_Emails)     
+- [Send Emails](Send_Emails)    
+- [List of those who starred and un-starred a repo](Stargazers)
```
Comment (vybhav72954) [inline JavaScript/README.md:26]: Follow the alphabetical order here as well. (I  know stargazers is the name, but acc to description you have provided it shouldn't be here)

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

**Comment by ireneelizabethsabu** [comment]
Sorted description in alphabetical order .Please review the PR @vybhav72954

File: JavaScript/README.md [hunk 2]
```diff
@@ -2,23 +2,30 @@
...
 # 📌 Javascript scripts associated with the project: 
 
+- [Active Pull Request Commentors](Active_pull_request_commentors)
 - [Amazon Review Scrapper](Amazon_Review_Scrapper)
 - [Automate Tinder](Automate_Tinder)
 - [Base64 Encoder Decoder](Base64_Encoder_Decoder)
 - [Blog Image Finder](Blog_Image_Finder)
 - [Brute Force Admin Panels](Brute_Force_Admin)
-- [Send bulk LinkedIn Requests](Bulk_LinkedIn_Requests)
+- [Bulk LinkedIn Requests sending](Bulk_LinkedIn_Requests)
```
Comment (vybhav72954) [inline JavaScript/README.md:11]: Lets keep to `Sender`

File: JavaScript/README.md [hunk 3]
```diff
@@ -2,23 +2,30 @@
...
 - [Citation Text Converter](Citation_Text_Converter)
 - [Cryptoprice](Cryptoprice)
 - [Download all Github repositories](Download_All_Repos)
```
Comment (vybhav72954) [inline JavaScript/README.md:14]: Lets keep to `Repositories`

File: JavaScript/README.md [hunk 4]
```diff
@@ -2,23 +2,30 @@
...
 - [Edit a Webpage](Edit_Webpage)
 - [Extract Code Comments](Extract_Comments)
 - [Follow automatically on Instagram](Follow_Instagram)
+- [Github Good First Issues extractor](Github_Good_First_Issues)
```
Comment (vybhav72954) [inline JavaScript/README.md:18]: `Github Good First Issues Extractor`

File: JavaScript/README.md [hunk 5]
```diff
@@ -2,23 +2,30 @@
...
 - [Github Graph Populator](GitHub_Graph_Populator)
 - [HTML to Markdown](Html_to_Markdown)
 - [Like Posts automatically on Instagram](Instagram_Like)
+- [List of those who starred and un-starred a repo](Stargazers)
```
Comment (vybhav72954) [inline JavaScript/README.md:22]: `GitHub Stargrazers`

File: JavaScript/README.md [hunk 6]
```diff
@@ -2,23 +2,30 @@
...
 - [ROT13 Decryption](ROT13_Decryption)
 - [RSA Key Pair Generator](RSA_Key_Pair_Generator)
 - [Send Emails](Send_Emails)     
+- [Twitter Common Followers for two users](Twitter_Followers_Overlap)
```
Comment (vybhav72954) [inline JavaScript/README.md:26]: `Twitter Mutual Followers`

File: JavaScript/README.md [hunk 7]
```diff
@@ -2,23 +2,30 @@
...
 - [Blog Image Finder](Blog_Image_Finder)
 - [Brute Force Admin Panels](Brute_Force_Admin)
-- [Send bulk LinkedIn Requests](Bulk_LinkedIn_Requests)
+- [Bulk LinkedIn Requests sending](Bulk_LinkedIn_Requests)
 - [Citation Text Converter](Citation_Text_Converter)
 - [Cryptoprice](Cryptoprice)
 - [Download all Github repositories](Download_All_Repos)
 - [Edit a Webpage](Edit_Webpage)
 - [Extract Code Comments](Extract_Comments)
 - [Follow automatically on Instagram](Follow_Instagram)
```
Comment (vybhav72954) [inline JavaScript/README.md:17]: `Instagram Follow Bot`

File: JavaScript/README.md [hunk 8]
```diff
@@ -2,23 +2,30 @@
...
+- [Github Good First Issues extractor](Github_Good_First_Issues)
 - [Github Graph Populator](GitHub_Graph_Populator)
 - [HTML to Markdown](Html_to_Markdown)
 - [Like Posts automatically on Instagram](Instagram_Like)
```
Comment (vybhav72954) [inline JavaScript/README.md:21]: `Instagram Like Bot`

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

**Comment by ireneelizabethsabu** [comment]
Please review the PR @vybhav72954

**Review by vybhav72954 — APPROVED** [review]

**Pull request merged**